src/ORM/OrmV2PersistenceStrategy.php

```diff
@@ -25,49 +28,42 @@
  */
 final class OrmV2PersistenceStrategy extends AbstractORMPersistenceStrategy
 {
-    public function inversedRelationshipMetadata(string $parent, string $child, string $field): ?InverseRelationshipMetadata
+    public function bidirectionalRelationshipMetadata(string $parent, string $child, string $field): ?RelationshipMetadata
```

Comment:
I had hard time to find a good name for the kind of relationship I want to grab metadata for...

it appears that we want all "bi-directional" relationships (well, not all actually, we don't need `ManyToMany` for which I havent found any problem)